src/marks/linearRegression.d.ts

```diff
@@ -2,23 +2,85 @@ import type {ChannelValue, ChannelValueDenseBinSpec, ChannelValueSpec} from "../
 import type {Data, MarkOptions, RenderableMark} from "../mark.js";
 import type {BinOptions, BinReducer} from "../transforms/bin.js";
 
+/** Options for the linearRegressionX and linearRegressionY marks. */
 interface LinearRegressionOptions extends MarkOptions, BinOptions {
+  /**
+   * A channel for the dependent variable **x**.
+   */
   x?: ChannelValueSpec;
+  /**
+   * A channel for the dependent variable **y**.
+   */
```

Comment:
these represent the dependent variable, and should probably go into their respective mark (x in linearRegressionX, and y in linearRegressionY)

Reply:
done